guides/common/modules/con_template-writing-reference.adoc

```diff
@@ -21,10 +21,8 @@ Smart Variables::
 For more information, see {MultiBaseURL}puppet_guide/chap-red_hat_satellite-puppet_guide-adding_puppet_modules_to_red_hat_satellite_6#sect-{Project_Link}-Puppet_Guide-Adding_Puppet_Modules_to_{Project_Link}_6-Configuring_Smart_Variables[Configuring Smart Variables] in the _Puppet Guide_.
 endif::[]
 
-ifdef::satellite[]
 Smart Class Parameters::
-For more information, see {MultiBaseURL}puppet_guide/chap-red_hat_satellite-puppet_guide-adding_puppet_modules_to_red_hat_satellite_6#sect-{Project_Link}-Puppet_Guide-Adding_Puppet_Modules_to_{Project_Link}_6-Configuring_Smart_Class_Parameters[Configuring Smart Class Parameters] in the _Puppet Guide_.
-endif::[]
+For more information, see {ManagingConfigurationsPuppetDocURL}Configuring_Puppet_Smart_Class_Parameters_managing-configurations-puppet[Configuring Puppet Smart Class Parameters] in _{ManagingConfigurationsPuppetDocTitle}_.
```

Comment:
Please wrap in `ifdef`s

Reply:
I did remove them since the document is actually available everywhere, rather than just satellite but I can add them back.

Reply:
Yes. The only reason they were previously hidden behind `ifdef` is that the Puppet guide wasn't upstreamed yet.

Reply:
In that case, all looks good :+1: merging